.github/workflows/main.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - name: Check compliance with isort
         run: isort -c .
       - name: Install Black
-        run: python -m pip install black==22.1.0
+        run: python -m pip install black==22.6.0
```

Comment:
Should we move this change to a separate PR?

Reply:
Sure